spec/unit/service_spec.rb

```diff
@@ -0,0 +1,140 @@
+require_relative 'spec_helper'
+include CernerSplunk::PathHelpers, CernerSplunk::PlatformHelpers, CernerSplunk::ServiceHelpers, CernerSplunk::RestartHelpers
```

Comment:
Ok, a comment here would be helpful then.  especially if we move all the other global namespace includes into the specs. (I was able to get all of them working except for this one with the include in the initial describe block)